docs/user-guide/getting-started.md

```diff
@@ -1,185 +1,197 @@
 ---
-    description: In this guide, we'll walk through the process of deploying Ambassador Edge Stack in Kubernetes for ingress routing.
+   description: In this guide, we'll walk through the process of deploying Ambassador Edge Stack in Kubernetes for ingress routing.
 ---
 # Quick Start Installation Guide
 
 In this guide, we'll walk you through installing and configuring the Ambassador Edge Stack in your Kubernetes cluster. Within a few minutes, your cluster will be routing HTTPS requests from the Internet to a backend service. You'll also have a sense of how the Ambassador Edge Stack is managed.
 
+Different options for installation include:
+
+* [Quick install (recommended!)](#quick-install)
```

Comment:
The quick install options needs be more prominent. Bigger font? A happy unicorn icon? Something that makes it really really clear that this option is the preferred option.